### ISSUE TITLE
Extra empty box in quick info

### ISSUE THREAD

**Issue by lxymahatma** [MrJul/ReSharper.EnhancedTooltip#179]
If i install EnhancedTooltip, the quick info is separated into many boxes and there's one extra empty box at the bottom.
Disable all the setting from Resharper->Options->Environment->IntelliSense->Enhanced Tooltip this problem still exist

Visual Studio version: 2022, 17.5.5
Resharper version: 2023.1.1
Enhanced Tooltip version: 3.20.0.2

variable and method when hovering over with Enhanced Tooltip installed (disabled all settings):
![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/66151783/36a7d1a4-af9d-4545-8cf2-eab693f47af9)
![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/66151783/d85be3b5-ac51-4a14-8e79-335d567841d1)

variable and method without Enhanced Tooltip installed:
![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/66151783/ec06b79d-b58f-485f-9974-b1673611c7a4)
![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/66151783/e043dc95-9ff1-45cc-9896-8f21c42d03ea)

**Comment by lxymahatma** [comment]
> 

Thanks for reply! Yeah disable codist remove the empty box 👍
Also because the quick into is separated into many boxes. And if i want to click the issue link, like the CS0219 in fourth box, when my cursor moves from the first box to the second box, the quick info just disappeared, so it's pretty hard to click the button, are there any solutions for this

**Comment by DimitarCC** [comment]
Yes. There is solution but someone must code it. Currently i cannot support anymore the project because of that #178

**Comment by lxymahatma** [comment]
> Yes. There is solution but someone must code it. Currently i cannot support anymore the project because of that #178

😂 i see, i hope you can get the license back soon
tho there might be something on some website (afaik they already made it) but it's not good to use and talk publicly🤣
Thanks for your reply!

**Comment by IneedHelp** [comment]
I have used Codist and Enhanced Tooltip side by side for years, but finally decided to ditch Enhanced Tooltip and ReSharper's tooltip decorations
![image](https://github.com/MrJul/ReSharper.EnhancedTooltip/assets/1366970/ad535ac7-896a-4f3c-9360-a7d5ef3068f6)
because it conflicts with functionality Codist is offering with Super Quick Info (symbol navigation and other features). Using both always gave me a bunch of empty boxes between metadata groups.

**Comment by IneedHelp** [comment]
With the new Codist 7.4 beta, both Codist and Enhanced Tooltips work together, except the space between those boxes makes it hard to get to a box other than the first one as mentioned here

> > 
> 
> Also because the quick into is separated into many boxes. And if i want to click the issue link, like the CS0219 in fourth box, when my cursor moves from the first box to the second box, the quick info just disappeared, so it's pretty hard to click the button, are there any solutions for this

so if there would be a way to remove the spaces between the boxes, it would be great, as Codist and Enhanced Tooltips could be used together.

**Comment by DimitarCC** [comment]
> Uppon further testing, both Codist and Enhanced Tooltips work together if _Replace Visual Studio tooltips_ in ReSharper Options is unchecked, except the space between those boxes makes it hard to get to a box other than the first one as mentioned here
> 
> > > 
> > 
> > 
> > Also because the quick into is separated into many boxes. And if i want to click the issue link, like the CS0219 in fourth box, when my cursor moves from the first box to the second box, the quick info just disappeared, so it's pretty hard to click the button, are there any solutions for this
> 
> so if there would be a way to remove the spaces between the boxes, it would be great, as Codist and Enhanced Tooltips could be used together.
> 
> More info about this here: [INFO: for people who use ReSharper with Enhanced Tooltips, but also want Codist #262](https://github.com/wmjordan/Codist/issues/262)

as i said everything is possible but somebody have to code it. ;)

**Comment by wmjordan** [comment]
> > Uppon further testing, both Codist and Enhanced Tooltips work together if _Replace Visual Studio tooltips_ in ReSharper Options is unchecked, except the space between those boxes makes it hard to get to a box other than the first one as mentioned here
> > > > 
> > > 
> > > 
> > > Also because the quick into is separated into many boxes. And if i want to click the issue link, like the CS0219 in fourth box, when my cursor moves from the first box to the second box, the quick info just disappeared, so it's pretty hard to click the button, are there any solutions for this
> > 
> > 
> > so if there would be a way to remove the spaces between the boxes, it would be great, as Codist and Enhanced Tooltips could be used together.
> > More info about this here: [INFO: for people who use ReSharper with Enhanced Tooltips, but also want Codist #262](https://github.com/wmjordan/Codist/issues/262)
> 
> as i said everything is possible but somebody have to code it. ;)

@DimitarCC 

@IneedHelp's issue brought me here.
I am so sorry to see that they are charging you such a whopping 200 euros/year license fee, instead of paying you and other contributors for this extension which adds great value to their product. What a shame!!!

**Comment by IneedHelp** [comment]
> 
> as i said everything is possible but somebody have to code it. ;)

I'm celebrating the fact that you got a license renewal then ;p

**Comment by DimitarCC** [comment]
I am planning to add real support for Codist's tooltip inside Enhanced tooltip. But that will take some time since i have to consolidate Resharper items with Codist items ;)

**Comment by wmjordan** [comment]
@DimitarCC 
Codist places a class named `Codist.QuickInfo.QuickInfoOverride+UIOverride` into the WPF Quick Info and used it to do the magic. You can find out that control and avoid wrapping the box around it.

**Comment by IneedHelp** [comment]
> I am planning to add real support for Codist's tooltip inside Enhanced tooltip. But that will take some time since i have to consolidate Resharper items with Codist items ;)

This would be amazing, I would love to be able to use the latest version of Codist with Enhanced tooltip, but for now I am sticking to Codist v7.4 8762 Beta, which is the last version that played nice with Enhanced tooltip. @DimitarCC thank you for taking the time to remove the gaps between info boxes, it makes it work better with Codist as we can actually use the interactive elements now.
